gax-grpc/src/test/java/com/google/api/gax/grpc/SettingsTest.java

```diff
@@ -268,77 +256,6 @@ public FakeStubSettings build() throws IOException {
     }
   }
 
-  private static class FakeSettings extends ClientSettings {
-
-    public UnaryCallSettings<Integer, Integer> fakeMethodSimple() {
-      return ((FakeStubSettings) getStubSettings()).fakeMethodSimple();
-    }
-
-    public PagedCallSettings<Integer, Integer, FakePagedListResponse> fakePagedMethod() {
-      return ((FakeStubSettings) getStubSettings()).fakePagedMethod();
-    }
-
-    public BatchingCallSettings<Integer, Integer> fakeMethodBatching() {
-      return ((FakeStubSettings) getStubSettings()).fakeMethodBatching();
-    }
-
-    public static Builder newBuilder() {
-      return Builder.createDefault();
-    }
-
-    public Builder toBuilder() {
-      return new Builder(this);
-    }
-
-    private FakeSettings(Builder settingsBuilder) throws IOException {
-      super(settingsBuilder);
-    }
-
-    private static class Builder extends ClientSettings.Builder {
-
-      protected Builder() throws IOException {
-        this((ClientContext) null);
-      }
-
-      protected Builder(ClientContext clientContext) {
-        super(FakeStubSettings.newBuilder(clientContext));
-      }
-
-      private static Builder createDefault() {
-        return new Builder(FakeStubSettings.newBuilder());
-      }
-
-      protected Builder(FakeSettings settings) {
-        super(settings.getStubSettings().toBuilder());
-      }
-
-      protected Builder(FakeStubSettings.Builder stubSettings) {
-        super(stubSettings);
-      }
-
-      public FakeStubSettings.Builder getStubSettingsBuilder() {
-        return ((FakeStubSettings.Builder) getStubSettings());
-      }
-
-      public UnaryCallSettings.Builder<Integer, Integer> fakeMethodSimple() {
-        return getStubSettingsBuilder().fakeMethodSimple();
-      }
-
-      public PagedCallSettings.Builder<Integer, Integer, FakePagedListResponse> fakePagedMethod() {
-        return getStubSettingsBuilder().fakePagedMethod();
-      }
-
-      public BatchingCallSettings.Builder<Integer, Integer> fakeMethodBatching() {
-        return getStubSettingsBuilder().fakeMethodBatching();
-      }
-
-      @Override
-      public FakeSettings build() throws IOException {
-        return new FakeSettings(this);
-      }
-    }
-  }
-
```

Comment:
I think these are used to simulate a gapic generated client

Reply:
Can you elaborate? It looks to me like this is simply testing that a fake is equal to a fake. I don't see where it tests any model code. What am I not seeing?

Reply:
`TestSettingsTest#callSettingsBuildFromTimeoutNoRetries()` seems like the only legit test here (that one which is not testing fake vs fake), and that one is not completely removed. So the cleanup looks legit to me.